### DISCUSSION TITLE
How and which external shuffle service can be used for a stackable operator?

### DISCUSSION THREAD

**Discussion by supsupsap** [stackabletech/spark-k8s-operator#390] [Q&A]
How and which external shuffle service can be used for a stackable operator? I want to use dynamic allocation. Is it possible?

**Comment by maltesander** [comment]
Hi @supsupsap,

there was a discussion about dynamic resource allocation here https://github.com/stackabletech/spark-k8s-operator/discussions/312

Does this help?

Kind regards,
Malte
